test/resources/eventlistener.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: listener
+spec:
+  serviceAccountName: tekton-triggers-example-sa
+  triggers:
+    - name: foo-trig
+      bindings:
+        - ref: pipeline-binding
+        - ref: message-binding
+      template:
+        name: pipeline-template
```

Comment:
I think we need a new line at the end.

Reply:
Why? It's created successfully and verified by the tests added in this pr.

Reply:
I see the red circle with a dash in it, so I think it's because of the absence of \n.
![Screenshot from 2020-08-08 13-31-42](https://user-images.githubusercontent.com/26519653/89705518-ad4e0e00-d97b-11ea-8c5d-cc69efa9076f.png)